jeremy2918/tsseract
# Fixed footer and user page

# Fix footer


This PR just fixes the footer and the user page of the application.

## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test` All tests should be passing successfully.
- `npm run dev`
- Check that the footer is in the right place at the bottom of the page.
- Go to the user page and check that it is responsive.
